.github/pull_request_template.md

```diff
@@ -8,4 +8,4 @@
 
 Resolve <!--- link the issue here -->
 
-## Screenshots (if appropriate):
+## Screenshots:
```

Comment:
This looks like totally unrelated to this PR.

Reply:
Totally agree, that unrelated things should be in separate PRs. No need to change it for this PR, but for the future it's better not to save the CI time that much.